src/database/sqlite/sqlite_health.c

```diff
@@ -55,15 +55,137 @@ static void sql_health_alarm_log_update(RRDHOST *host, ALARM_ENTRY *ae)
 }
 
 /* Health related SQL queries
-   Inserts an entry in the table
+ *
+ * Inserts an entry in the tables
+ *   alert_queue
+ *   health_log
+ *   health_log_detail
+ *
 */
 
+int calculate_delay(RRDCALC_STATUS old_status, RRDCALC_STATUS new_status)
+{
+    int delay = 0;
+    switch(old_status) {
+        case RRDCALC_STATUS_REMOVED:
+            switch (new_status) {
+                case RRDCALC_STATUS_UNINITIALIZED:
+                    delay = 600;
```

Comment:
Probably we could add some `defines` for the magic numbers `600`, `10` and `0`.

Reply:
I will add (these may be refined in the future)